Application/src/main/java/model/Valutazione/ValutazioneDAO.java

```diff
@@ -0,0 +1,84 @@
+package model.Valutazione;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.sql.ResultSet;
+import java.util.Collection;
+import java.util.ArrayList;
+
+import model.ConDB;
+
+public class ValutazioneDAO {
+    public synchronized void save(ValutazioneBean valutazione) throws SQLException {
```

Comment:
Vedi come abbiamo fatto io, carmine e andrea per il try (ovvero aggiungi il finally) perché ConDB è un pool di connessioni che le salva in una lista facendo ConDB.releaseConnection invece di chiuderle, se la chiudi poi ne si viene a creare un'altra successivamente e si perde il vantaggio del pool di connessioni